apiv2/api/openapi/openapi.yaml

```diff
@@ -3777,6 +3909,26 @@ components:
                     description: The node type
                     format: enum
             description: A node in the location tree.
+        ListOSUpdatePolicyResponse:
+            required:
+                - osUpdatePolicies
+                - totalElements
+                - hasNext
+            type: object
+            properties:
+                osUpdatePolicies:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/OSUpdatePolicy'
+                    description: Sorted and filtered list of os update policies.
```

Comment:
extreme nit: consistency between OS Update Policies and os update policies across the PR.